From 202e8c3bab0792d3db71fbc7699a2219d2ca5a59 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 26 Mar 2012 20:58:47 +0200
Subject: [PATCH 106/125] journalctl,loginctl: drop systemd- prefix in binary
 names
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Let's make things a bit easier to type, drop the systemd- prefix for
journalctl and loginctl, but provide the old names for compat.

All systemd binaries are hence now prefixed with "systemd-" with the
exception of the three primary user interface binaries:

systemctl
loginctl
journalctl

For those three we do provide systemd-xyz names as well, via symlinks:

systemd-systemctl → systemctl
systemd-loginctl → loginctl
systemd-journalctl → journalctl

We do this only for the *primary* user tools, in order to avoid
unnecessary namespace problems. That means tools like systemd-notify
stay the way they are.

Conflicts:

	TODO
---
 Makefile.am                   |   56 +++--
 man/journalctl.xml            |  253 +++++++++++++++++++++++
 man/loginctl.xml              |  458 +++++++++++++++++++++++++++++++++++++++++
 man/systemctl.xml             |    7 +-
 man/systemd-journalctl.xml    |  252 -----------------------
 man/systemd-journald.conf.xml |    2 +-
 man/systemd-loginctl.xml      |  457 ----------------------------------------
 man/systemd-logind.conf.xml   |    2 +-
 man/systemd.exec.xml          |    3 +-
 9 files changed, 762 insertions(+), 728 deletions(-)
 create mode 100644 man/journalctl.xml
 create mode 100644 man/loginctl.xml
 delete mode 100644 man/systemd-journalctl.xml
 delete mode 100644 man/systemd-loginctl.xml

diff --git a/Makefile.am b/Makefile.am
index 079c118..9b43e9f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -695,16 +695,22 @@ MANPAGES = \
 	man/systemd-cat.1 \
         man/systemd-machine-id-setup.1 \
         man/systemd-journald.conf.5 \
-        man/systemd-journalctl.1
+        man/journalctl.1
 
 MANPAGES_ALIAS = \
 	man/reboot.8 \
 	man/poweroff.8 \
-	man/init.1
+	man/init.1 \
+        man/systemd-systemctl.1 \
+        man/systemd-loginctl.1 \
+        man/systemd-journalctl.1
 
 man/reboot.8: man/halt.8
 man/poweroff.8: man/halt.8
 man/init.1: man/systemd.1
+man/systemd-systemctl.1: man/systemctl.1
+man/systemd-loginctl.1: man/loginctl.1
+man/systemd-journalctl.1: man/journalctl.1
 
 XML_FILES = \
 	${patsubst %.1,%.xml,${patsubst %.3,%.xml,${patsubst %.5,%.xml,${patsubst %.7,%.xml,${patsubst %.8,%.xml,$(MANPAGES)}}}}}
@@ -984,6 +990,14 @@ systemctl_LDADD = \
 	libsystemd-id128.la \
 	$(DBUS_LIBS)
 
+systemctl-install-hook:
+	cd $(DESTDIR)$(rootbindir) && \
+		rm -f systemd-systemctl && \
+		$(LN_S) systemctl systemd-systemctl
+
+INSTALL_EXEC_HOOKS += \
+        systemctl-install-hook
+
 systemd_notify_SOURCES = \
 	src/notify.c \
 	src/readahead/sd-readahead.c
@@ -1258,26 +1272,34 @@ systemd_cat_LDADD = \
 	libsystemd-basic.la \
 	libsystemd-journal.la
 
-systemd_journalctl_SOURCES = \
+journalctl_SOURCES = \
 	src/journal/journalctl.c \
 	src/pager.c \
 	src/logs-show.c
 
-systemd_journalctl_LDADD = \
+journalctl_LDADD = \
 	libsystemd-basic.la \
 	libsystemd-journal.la \
 	libsystemd-id128.la
 
 if HAVE_XZ
-systemd_journalctl_SOURCES += \
+journalctl_SOURCES += \
 	src/journal/compress.c
-systemd_journalctl_CFLAGS = \
+journalctl_CFLAGS = \
 	$(AM_CFLAGS) \
 	$(XZ_CFLAGS)
-systemd_journalctl_LDADD += \
+journalctl_LDADD += \
 	$(XZ_LIBS)
 endif
 
+journalctl-install-hook:
+	cd $(DESTDIR)$(rootbindir) && \
+		rm -f systemd-journalctl && \
+		$(LN_S) journalctl systemd-journalctl
+
+INSTALL_EXEC_HOOKS += \
+        journalctl-install-hook
+
 test_journal_SOURCES = \
 	src/journal/test-journal.c \
 	src/journal/sd-journal.c \
@@ -1364,7 +1386,7 @@ rootlibexec_PROGRAMS += \
 	systemd-journald
 
 rootbin_PROGRAMS += \
-	systemd-journalctl
+	journalctl
 
 bin_PROGRAMS += \
 	systemd-cat
@@ -1864,7 +1886,7 @@ rootlibexec_PROGRAMS += \
 	systemd-logind \
 	systemd-user-sessions
 
-systemd_loginctl_SOURCES = \
+loginctl_SOURCES = \
 	src/login/loginctl.c \
 	src/login/sysfs-show.c \
 	src/dbus-common.c \
@@ -1872,18 +1894,26 @@ systemd_loginctl_SOURCES = \
 	src/cgroup-util.c \
 	src/pager.c
 
-systemd_loginctl_CFLAGS = \
+loginctl_CFLAGS = \
 	$(AM_CFLAGS) \
 	$(DBUS_CFLAGS) \
 	$(UDEV_CFLAGS)
 
-systemd_loginctl_LDADD = \
+loginctl_LDADD = \
 	libsystemd-basic.la \
 	$(DBUS_LIBS) \
 	$(UDEV_LIBS)
 
 rootbin_PROGRAMS += \
-	systemd-loginctl
+	loginctl
+
+loginctl-install-hook:
+	cd $(DESTDIR)$(rootbindir) && \
+		rm -f systemd-loginctl && \
+		$(LN_S) loginctl systemd-loginctl
+
+INSTALL_EXEC_HOOKS += \
+        loginctl-install-hook
 
 test_login_SOURCES = \
 	src/login/test-login.c
@@ -2048,7 +2078,7 @@ nodist_udevrules_DATA += \
 MANPAGES += \
 	man/systemd-logind.conf.5 \
 	man/sd-login.7 \
-	man/systemd-loginctl.1 \
+	man/loginctl.1 \
 	man/sd_login_monitor_new.3 \
 	man/sd_pid_get_session.3 \
 	man/sd_uid_get_state.3 \
diff --git a/man/journalctl.xml b/man/journalctl.xml
new file mode 100644
index 0000000..c4d2a7e
--- /dev/null
+++ b/man/journalctl.xml
@@ -0,0 +1,253 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
+        "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+
+<!--
+  This file is part of systemd.
+
+  Copyright 2012 Lennart Poettering
+
+  systemd is free software; you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  systemd is distributed in the hope that it will be useful, but
+  WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with systemd; If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<refentry id="journalctl">
+
+        <refentryinfo>
+                <title>journalctl</title>
+                <productname>systemd</productname>
+
+                <authorgroup>
+                        <author>
+                                <contrib>Developer</contrib>
+                                <firstname>Lennart</firstname>
+                                <surname>Poettering</surname>
+                                <email>lennart@poettering.net</email>
+                        </author>
+                </authorgroup>
+        </refentryinfo>
+
+        <refmeta>
+                <refentrytitle>journalctl</refentrytitle>
+                <manvolnum>1</manvolnum>
+        </refmeta>
+
+        <refnamediv>
+                <refname>journalctl</refname>
+                <refname>systemd-journalctl</refname>
+                <refpurpose>Query the systemd journal</refpurpose>
+        </refnamediv>
+
+        <refsynopsisdiv>
+                <cmdsynopsis>
+                        <command>journalctl <arg choice="opt" rep="repeat">OPTIONS</arg> <arg choice="opt">MATCH</arg></command>
+                </cmdsynopsis>
+        </refsynopsisdiv>
+
+        <refsect1>
+                <title>Description</title>
+
+                <para><command>journalctl</command> may be
+                used to query the contents of the
+                <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+                journal.</para>
+
+                <para>If called without parameter will show the full
+                contents of the journal, starting with the oldest
+                entry collected.</para>
+
+                <para>If a match argument is passed the output is
+                filtered accordingly. A match is in the format
+                <literal>FIELD=VALUE</literal>,
+                e.g. <literal>_SYSTEMD_UNIT=httpd.service</literal>.</para>
+
+                <para>Output is interleaved from all accessible
+                journal files, whether they are rotated or currently
+                being written, and regardless whether they belong to the
+                system itself or are accessible user journals.</para>
+
+                <para>All users are granted access to their private
+                per-user journals. However, by default only root and
+                users who are members of the <literal>adm</literal>
+                group get access to the system journal and the
+                journals of other users.</para>
+        </refsect1>
+
+        <refsect1>
+                <title>Options</title>
+
+                <para>The following options are understood:</para>
+
+                <variablelist>
+                        <varlistentry>
+                                <term><option>--help</option></term>
+                                <term><option>-h</option></term>
+
+                                <listitem><para>Prints a short help
+                                text and exits.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--version</option></term>
+
+                                <listitem><para>Prints a short version
+                                string and exits.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--no-pager</option></term>
+
+				<listitem><para>Do not pipe output into a
+				pager.</para></listitem>
+			</varlistentry>
+
+                        <varlistentry>
+                                <term><option>--all</option></term>
+                                <term><option>-a</option></term>
+
+                                <listitem><para>Show all fields in
+                                full, even if they include unprintable
+                                characters or are very
+                                long.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--follow</option></term>
+                                <term><option>-f</option></term>
+
+                                <listitem><para>Show only most recent
+                                journal entries, and continously print
+                                new entries as they are appended to
+                                the journal.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--lines=</option></term>
+                                <term><option>-n</option></term>
+
+                                <listitem><para>Controls the number of
+                                journal lines to show, counting from
+                                the most recent ones. Takes a positive
+                                integer argument. In follow mode
+                                defaults to 10, otherwise is unset
+                                thus not limiting how many lines are
+                                shown.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--no-tail</option></term>
+
+                                <listitem><para>Show all stored output
+                                lines, even in follow mode. Undoes the
+                                effect of
+                                <option>--lines=</option>.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--output=</option></term>
+                                <term><option>-o</option></term>
+
+                                <listitem><para>Controls the
+                                formatting of the journal entries that are
+                                shown. Takes one of
+                                <literal>short</literal>,
+                                <literal>short-monotonic</literal>,
+                                <literal>verbose</literal>,
+                                <literal>export</literal>,
+                                <literal>json</literal>,
+                                <literal>cat</literal>. <literal>short</literal>
+                                is the default and generates an output
+                                that is mostly identical to the
+                                formatting of classic syslog log
+                                files, showing one line per journal
+                                entry. <literal>short-monotonic</literal>
+                                is very similar but shows monotonic
+                                timestamps instead of wallclock
+                                timestamps. <literal>verbose</literal>
+                                shows the full structered entry items
+                                with all
+                                fiels. <literal>export</literal>
+                                serializes the journal into a binary
+                                (but mostly text-based) stream
+                                suitable for backups and network
+                                transfer. <literal>json</literal>
+                                formats entries as JSON data
+                                structures. <literal>cat</literal>
+                                generates a very terse output only
+                                showing the actual message of each
+                                journal entry with no meta data, not
+                                even a timestamp.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--quiet</option></term>
+                                <term><option>-q</option></term>
+
+                                <listitem><para>Suppresses any warning
+                                message regarding inaccessable system
+                                journals when run as normal
+                                user.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--new-id128</option></term>
+
+                                <listitem><para>Instead of showing
+                                journal contents generate a new 128
+                                bit ID suitable for identifying
+                                messages. This is intended for usage
+                                by developers who need a new
+                                identifier for a new message they
+                                introduce and want to make
+                                recognizable. Will print the new ID in
+                                three different formats which can be
+                                copied into source code or
+                                similar.</para></listitem>
+                        </varlistentry>
+
+                </variablelist>
+        </refsect1>
+
+        <refsect1>
+                <title>Exit status</title>
+
+                <para>On success 0 is returned, a non-zero failure
+                code otherwise.</para>
+        </refsect1>
+
+        <refsect1>
+                <title>Environment</title>
+
+                <variablelist>
+                        <varlistentry>
+                                <term><varname>$SYSTEMD_PAGER</varname></term>
+                                <listitem><para>Pager to use when
+                                <option>--no-pager</option> is not given;
+                                overrides <varname>$PAGER</varname>.  Setting
+                                this to an empty string or the value
+                                <literal>cat</literal> is equivalent to passing
+                                <option>--no-pager</option>.</para></listitem>
+                        </varlistentry>
+                </variablelist>
+        </refsect1>
+
+        <refsect1>
+                <title>See Also</title>
+                <para>
+                        <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>systemd-journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                </para>
+        </refsect1>
+
+</refentry>
diff --git a/man/loginctl.xml b/man/loginctl.xml
new file mode 100644
index 0000000..be72cc3
--- /dev/null
+++ b/man/loginctl.xml
@@ -0,0 +1,458 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
+        "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+
+<!--
+  This file is part of systemd.
+
+  Copyright 2010 Lennart Poettering
+
+  systemd is free software; you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  systemd is distributed in the hope that it will be useful, but
+  WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with systemd; If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<refentry id="loginctl">
+
+        <refentryinfo>
+                <title>loginctl</title>
+                <productname>systemd</productname>
+
+                <authorgroup>
+                        <author>
+                                <contrib>Developer</contrib>
+                                <firstname>Lennart</firstname>
+                                <surname>Poettering</surname>
+                                <email>lennart@poettering.net</email>
+                        </author>
+                </authorgroup>
+        </refentryinfo>
+
+        <refmeta>
+                <refentrytitle>loginctl</refentrytitle>
+                <manvolnum>1</manvolnum>
+        </refmeta>
+
+        <refnamediv>
+                <refname>loginctl</refname>
+                <refname>systemd-loginctl</refname>
+                <refpurpose>Control the systemd login manager</refpurpose>
+        </refnamediv>
+
+        <refsynopsisdiv>
+                <cmdsynopsis>
+                        <command>loginctl <arg choice="opt" rep="repeat">OPTIONS</arg> <arg choice="req">COMMAND</arg> <arg choice="opt" rep="repeat">NAME</arg></command>
+                </cmdsynopsis>
+        </refsynopsisdiv>
+
+        <refsect1>
+                <title>Description</title>
+
+                <para><command>loginctl</command> may be used to
+                introspect and control the state of the
+                <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+                login manager.</para>
+        </refsect1>
+
+        <refsect1>
+                <title>Options</title>
+
+                <para>The following options are understood:</para>
+
+                <variablelist>
+                        <varlistentry>
+                                <term><option>--help</option></term>
+                                <term><option>-h</option></term>
+
+                                <listitem><para>Prints a short help
+                                text and exits.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--version</option></term>
+
+                                <listitem><para>Prints a short version
+                                string and exits.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--property=</option></term>
+                                <term><option>-p</option></term>
+
+                                <listitem><para>When showing
+                                session/user/ properties, limit
+                                display to certain properties as
+                                specified as argument. If not
+                                specified all set properties are
+                                shown. The argument should be a
+                                property name, such as
+                                <literal>Sessions</literal>. If
+                                specified more than once all
+                                properties with the specified names
+                                are shown.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--all</option></term>
+                                <term><option>-a</option></term>
+
+                                <listitem><para>When showing
+                                unit/job/manager properties, show all
+                                properties regardless whether they are
+                                set or not.</para></listitem>
+                        </varlistentry>
+
+
+                        <varlistentry>
+                                <term><option>--no-pager</option></term>
+
+				<listitem><para>Do not pipe output into a
+				pager.</para></listitem>
+			</varlistentry>
+
+                        <varlistentry>
+                                <term><option>--kill-who=</option></term>
+
+                                <listitem><para>When used with
+                                <command>kill-session</command>,
+                                choose which processes to kill. Must
+                                be one of <option>leader</option>, or
+                                <option>all</option> to select whether
+                                to kill only the leader process of the
+                                session or all processes of the
+                                session. If omitted defaults to
+                                <option>all</option>.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>--signal=</option></term>
+                                <term><option>-s</option></term>
+
+                                <listitem><para>When used with
+                                <command>kill-session</command> or
+                                <command>kill-user</command>, choose
+                                which signal to send to selected
+                                processes. Must be one of the well
+                                known signal specifiers such as
+                                SIGTERM, SIGINT or SIGSTOP. If omitted
+                                defaults to
+                                <option>SIGTERM</option>.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>-H</option></term>
+                                <term><option>--host</option></term>
+
+                                <listitem><para>Execute operation
+                                remotely. Specify a hostname, or
+                                username and hostname separated by @,
+                                to connect to. This will use SSH to
+                                talk to the remote login manager
+                                instance.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><option>-P</option></term>
+                                <term><option>--privileged</option></term>
+
+                                <listitem><para>Acquire privileges via
+                                PolicyKit before executing the
+                                operation.</para></listitem>
+                        </varlistentry>
+                </variablelist>
+
+                <para>The following commands are understood:</para>
+
+                <variablelist>
+                        <varlistentry>
+                                <term><command>list-sessions</command></term>
+
+                                <listitem><para>List current sessions.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>session-status [ID...]</command></term>
+
+                                <listitem><para>Show terse runtime
+                                status information about one or more
+                                sessions. This function is intended to
+                                generate human-readable output. If you
+                                are looking for computer-parsable
+                                output, use
+                                <command>show-session</command>
+                                instead.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>show-session [ID...]</command></term>
+
+                                <listitem><para>Show properties of one
+                                or more sessions or the manager
+                                itself. If no argument is specified
+                                properties of the manager will be
+                                shown. If a session ID is specified
+                                properties of the session is shown. By
+                                default, empty properties are
+                                suppressed. Use <option>--all</option>
+                                to show those too. To select specific
+                                properties to show use
+                                <option>--property=</option>. This
+                                command is intended to be used
+                                whenever computer-parsable output is
+                                required. Use
+                                <command>session-status</command> if
+                                you are looking for formatted
+                                human-readable
+                                output.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>activate [ID...]</command></term>
+
+                                <listitem><para>Activate one or more
+                                sessions. This brings one or more
+                                sessions into the foreground, if
+                                another session is currently in the
+                                foreground on the respective
+                                seat.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>lock-session [ID...]</command></term>
+                                <term><command>unlock-session [ID...]</command></term>
+
+                                <listitem><para>Activates/deactivates
+                                the screen lock on one or more
+                                sessions, if the session supports it.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>terminate-session [ID...]</command></term>
+
+                                <listitem><para>Terminates a
+                                session. This kills all processes of
+                                the session and deallocates all
+                                resources attached to the
+                                session.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>kill-session [ID...]</command></term>
+
+                                <listitem><para>Send a signal to one
+                                or more processes of the session. Use
+                                <option>--kill-who=</option> to select
+                                which process to kill. Use
+                                <option>--signal=</option> to select
+                                the signal to send.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>list-users</command></term>
+
+                                <listitem><para>List currently logged
+                                in users.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>user-status [USER...]</command></term>
+
+                                <listitem><para>Show terse runtime
+                                status information about one or more
+                                logged in users. This function is
+                                intended to generate human-readable
+                                output. If you are looking for
+                                computer-parsable output, use
+                                <command>show-user</command>
+                                instead. Users may be specified by
+                                their usernames or numeric user
+                                IDs.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>show-user [USER...]</command></term>
+
+                                <listitem><para>Show properties of one
+                                or more users or the manager
+                                itself. If no argument is specified
+                                properties of the manager will be
+                                shown. If a user is specified
+                                properties of the user is shown. By
+                                default, empty properties are
+                                suppressed. Use <option>--all</option>
+                                to show those too. To select specific
+                                properties to show use
+                                <option>--property=</option>. This
+                                command is intended to be used
+                                whenever computer-parsable output is
+                                required. Use
+                                <command>user-status</command> if
+                                you are looking for formatted
+                                human-readable
+                                output.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>enable-linger [USER...]</command></term>
+                                <term><command>disable-linger [USER...]</command></term>
+
+                                <listitem><para>Enable/disable user
+                                lingering for one or more users. If
+                                enabled for a specific user a user
+                                manager is spawned for him/her at
+                                boot, and kept around after
+                                logouts. This allows users who aren't
+                                logged in to run long-running
+                                services.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>terminate-user [USER...]</command></term>
+
+                                <listitem><para>Terminates all
+                                sessions of a user. This kills all
+                                processes of all sessions of the user
+                                and deallocates all runtime resources
+                                attached to the
+                                user.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>kill-user [USER...]</command></term>
+
+                                <listitem><para>Send a signal to all
+                                processes of a user. Use
+                                <option>--signal=</option> to select
+                                the signal to send.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>list-seats</command></term>
+
+                                <listitem><para>List currently
+                                available seats on the local
+                                system.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>seat-status [NAME...]</command></term>
+
+                                <listitem><para>Show terse runtime
+                                status information about one or more
+                                seats. This function is
+                                intended to generate human-readable
+                                output. If you are looking for
+                                computer-parsable output, use
+                                <command>show-seat</command>
+                                instead.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>show-seat [NAME...]</command></term>
+
+                                <listitem><para>Show properties of one
+                                or more seats or the manager
+                                itself. If no argument is specified
+                                properties of the manager will be
+                                shown. If a seat is specified
+                                properties of the seat are shown. By
+                                default, empty properties are
+                                suppressed. Use <option>--all</option>
+                                to show those too. To select specific
+                                properties to show use
+                                <option>--property=</option>. This
+                                command is intended to be used
+                                whenever computer-parsable output is
+                                required. Use
+                                <command>seat-status</command> if you
+                                are looking for formatted
+                                human-readable
+                                output.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>attach [NAME] [DEVICE...]</command></term>
+
+                                <listitem><para>Attach one or more
+                                devices to a seat. The devices should
+                                be specified via device paths in the
+                                <filename>/sys</filename> file
+                                system. To create a new seat attach at
+                                least one graphics card to a
+                                previously unused seat names. seat
+                                names may consist only of a-z, A-Z,
+                                0-9, "-" and "_" and must be prefixed
+                                with "seat". To drop assignment of a
+                                device to a specific seat just
+                                reassign it to a different seat, or
+                                use
+                                <command>flush-devices</command>.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>flush-devices</command></term>
+
+                                <listitem><para>Removes all device
+                                assignments previously created with
+                                <command>attach</command>. After this
+                                call only automatically generated
+                                seats will remain and all seat
+                                hardware is assigned to
+                                them.</para></listitem>
+                        </varlistentry>
+
+                        <varlistentry>
+                                <term><command>terminate-seat [NAME...]</command></term>
+
+                                <listitem><para>Terminates all
+                                sessions on a seat. This kills all
+                                processes of all sessions on a seat and
+                                deallocates all runtime resources
+                                attached to them.</para></listitem>
+                        </varlistentry>
+                </variablelist>
+
+        </refsect1>
+
+        <refsect1>
+                <title>Exit status</title>
+
+                <para>On success 0 is returned, a non-zero failure
+                code otherwise.</para>
+        </refsect1>
+
+        <refsect1>
+                <title>Environment</title>
+
+                <variablelist>
+                        <varlistentry>
+                                <term><varname>$SYSTEMD_PAGER</varname></term>
+                                <listitem><para>Pager to use when
+                                <option>--no-pager</option> is not given;
+                                overrides <varname>$PAGER</varname>.  Setting
+                                this to an empty string or the value
+                                <literal>cat</literal> is equivalent to passing
+                                <option>--no-pager</option>.</para></listitem>
+                        </varlistentry>
+                </variablelist>
+        </refsect1>
+
+        <refsect1>
+                <title>See Also</title>
+                <para>
+                        <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>systemd-logind.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                </para>
+        </refsect1>
+
+</refentry>
diff --git a/man/systemctl.xml b/man/systemctl.xml
index c467ed8..ffe0164 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -44,6 +44,7 @@
 
         <refnamediv>
                 <refname>systemctl</refname>
+                <refname>systemd-systemctl</refname>
                 <refpurpose>Control the systemd system and service manager</refpurpose>
         </refnamediv>
 
@@ -448,7 +449,7 @@
                                 formatting of the journal entries that
                                 are shown. For the available choices
                                 see
-                                <citerefentry><refentrytitle>systemd-journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>. Defaults
+                                <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>. Defaults
                                 to
                                 <literal>short</literal>.</para></listitem>
                         </varlistentry>
@@ -1179,8 +1180,8 @@
                 <para>
                         <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>systemadm</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemd-journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemd-loginctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>loginctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>wall</refentrytitle><manvolnum>1</manvolnum></citerefentry>
diff --git a/man/systemd-journalctl.xml b/man/systemd-journalctl.xml
deleted file mode 100644
index 1441eca..0000000
--- a/man/systemd-journalctl.xml
+++ /dev/null
@@ -1,252 +0,0 @@
-<?xml version='1.0'?> <!--*-nxml-*-->
-<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
-        "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
-
-<!--
-  This file is part of systemd.
-
-  Copyright 2012 Lennart Poettering
-
-  systemd is free software; you can redistribute it and/or modify it
-  under the terms of the GNU General Public License as published by
-  the Free Software Foundation; either version 2 of the License, or
-  (at your option) any later version.
-
-  systemd is distributed in the hope that it will be useful, but
-  WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-  General Public License for more details.
-
-  You should have received a copy of the GNU General Public License
-  along with systemd; If not, see <http://www.gnu.org/licenses/>.
--->
-
-<refentry id="systemd-journalctl">
-
-        <refentryinfo>
-                <title>systemd-journalctl</title>
-                <productname>systemd</productname>
-
-                <authorgroup>
-                        <author>
-                                <contrib>Developer</contrib>
-                                <firstname>Lennart</firstname>
-                                <surname>Poettering</surname>
-                                <email>lennart@poettering.net</email>
-                        </author>
-                </authorgroup>
-        </refentryinfo>
-
-        <refmeta>
-                <refentrytitle>systemd-journalctl</refentrytitle>
-                <manvolnum>1</manvolnum>
-        </refmeta>
-
-        <refnamediv>
-                <refname>systemd-journalctl</refname>
-                <refpurpose>Query the systemd journal</refpurpose>
-        </refnamediv>
-
-        <refsynopsisdiv>
-                <cmdsynopsis>
-                        <command>systemd-journalctl <arg choice="opt" rep="repeat">OPTIONS</arg> <arg choice="opt">MATCH</arg></command>
-                </cmdsynopsis>
-        </refsynopsisdiv>
-
-        <refsect1>
-                <title>Description</title>
-
-                <para><command>systemd-journalctl</command> may be
-                used to query the contents of the
-                <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-                journal.</para>
-
-                <para>If called without parameter will show the full
-                contents of the journal, starting with the oldest
-                entry collected.</para>
-
-                <para>If a match argument is passed the output is
-                filtered accordingly. A match is in the format
-                <literal>FIELD=VALUE</literal>,
-                e.g. <literal>_SYSTEMD_UNIT=httpd.service</literal>.</para>
-
-                <para>Output is interleaved from all accessible
-                journal files, whether they are rotated or currently
-                being written, and regardless whether they belong to the
-                system itself or are accessible user journals.</para>
-
-                <para>All users are granted access to their private
-                per-user journals. However, by default only root and
-                users who are members of the <literal>adm</literal>
-                group get access to the system journal and the
-                journals of other users.</para>
-        </refsect1>
-
-        <refsect1>
-                <title>Options</title>
-
-                <para>The following options are understood:</para>
-
-                <variablelist>
-                        <varlistentry>
-                                <term><option>--help</option></term>
-                                <term><option>-h</option></term>
-
-                                <listitem><para>Prints a short help
-                                text and exits.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--version</option></term>
-
-                                <listitem><para>Prints a short version
-                                string and exits.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--no-pager</option></term>
-
-				<listitem><para>Do not pipe output into a
-				pager.</para></listitem>
-			</varlistentry>
-
-                        <varlistentry>
-                                <term><option>--all</option></term>
-                                <term><option>-a</option></term>
-
-                                <listitem><para>Show all fields in
-                                full, even if they include unprintable
-                                characters or are very
-                                long.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--follow</option></term>
-                                <term><option>-f</option></term>
-
-                                <listitem><para>Show only most recent
-                                journal entries, and continously print
-                                new entries as they are appended to
-                                the journal.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--lines=</option></term>
-                                <term><option>-n</option></term>
-
-                                <listitem><para>Controls the number of
-                                journal lines to show, counting from
-                                the most recent ones. Takes a positive
-                                integer argument. In follow mode
-                                defaults to 10, otherwise is unset
-                                thus not limiting how many lines are
-                                shown.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--no-tail</option></term>
-
-                                <listitem><para>Show all stored output
-                                lines, even in follow mode. Undoes the
-                                effect of
-                                <option>--lines=</option>.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--output=</option></term>
-                                <term><option>-o</option></term>
-
-                                <listitem><para>Controls the
-                                formatting of the journal entries that are
-                                shown. Takes one of
-                                <literal>short</literal>,
-                                <literal>short-monotonic</literal>,
-                                <literal>verbose</literal>,
-                                <literal>export</literal>,
-                                <literal>json</literal>,
-                                <literal>cat</literal>. <literal>short</literal>
-                                is the default and generates an output
-                                that is mostly identical to the
-                                formatting of classic syslog log
-                                files, showing one line per journal
-                                entry. <literal>short-monotonic</literal>
-                                is very similar but shows monotonic
-                                timestamps instead of wallclock
-                                timestamps. <literal>verbose</literal>
-                                shows the full structered entry items
-                                with all
-                                fiels. <literal>export</literal>
-                                serializes the journal into a binary
-                                (but mostly text-based) stream
-                                suitable for backups and network
-                                transfer. <literal>json</literal>
-                                formats entries as JSON data
-                                structures. <literal>cat</literal>
-                                generates a very terse output only
-                                showing the actual message of each
-                                journal entry with no meta data, not
-                                even a timestamp.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--quiet</option></term>
-                                <term><option>-q</option></term>
-
-                                <listitem><para>Suppresses any warning
-                                message regarding inaccessable system
-                                journals when run as normal
-                                user.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--new-id128</option></term>
-
-                                <listitem><para>Instead of showing
-                                journal contents generate a new 128
-                                bit ID suitable for identifying
-                                messages. This is intended for usage
-                                by developers who need a new
-                                identifier for a new message they
-                                introduce and want to make
-                                recognizable. Will print the new ID in
-                                three different formats which can be
-                                copied into source code or
-                                similar.</para></listitem>
-                        </varlistentry>
-
-                </variablelist>
-        </refsect1>
-
-        <refsect1>
-                <title>Exit status</title>
-
-                <para>On success 0 is returned, a non-zero failure
-                code otherwise.</para>
-        </refsect1>
-
-        <refsect1>
-                <title>Environment</title>
-
-                <variablelist>
-                        <varlistentry>
-                                <term><varname>$SYSTEMD_PAGER</varname></term>
-                                <listitem><para>Pager to use when
-                                <option>--no-pager</option> is not given;
-                                overrides <varname>$PAGER</varname>.  Setting
-                                this to an empty string or the value
-                                <literal>cat</literal> is equivalent to passing
-                                <option>--no-pager</option>.</para></listitem>
-                        </varlistentry>
-                </variablelist>
-        </refsect1>
-
-        <refsect1>
-                <title>See Also</title>
-                <para>
-                        <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemd-journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-                </para>
-        </refsect1>
-
-</refentry>
diff --git a/man/systemd-journald.conf.xml b/man/systemd-journald.conf.xml
index f3cd4db..37dae68 100644
--- a/man/systemd-journald.conf.xml
+++ b/man/systemd-journald.conf.xml
@@ -246,7 +246,7 @@
                   <title>See Also</title>
                   <para>
                           <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd-journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
                   </para>
         </refsect1>
diff --git a/man/systemd-loginctl.xml b/man/systemd-loginctl.xml
deleted file mode 100644
index 6a28276..0000000
--- a/man/systemd-loginctl.xml
+++ /dev/null
@@ -1,457 +0,0 @@
-<?xml version='1.0'?> <!--*-nxml-*-->
-<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
-        "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
-
-<!--
-  This file is part of systemd.
-
-  Copyright 2010 Lennart Poettering
-
-  systemd is free software; you can redistribute it and/or modify it
-  under the terms of the GNU General Public License as published by
-  the Free Software Foundation; either version 2 of the License, or
-  (at your option) any later version.
-
-  systemd is distributed in the hope that it will be useful, but
-  WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-  General Public License for more details.
-
-  You should have received a copy of the GNU General Public License
-  along with systemd; If not, see <http://www.gnu.org/licenses/>.
--->
-
-<refentry id="systemd-loginctl">
-
-        <refentryinfo>
-                <title>systemd-loginctl</title>
-                <productname>systemd</productname>
-
-                <authorgroup>
-                        <author>
-                                <contrib>Developer</contrib>
-                                <firstname>Lennart</firstname>
-                                <surname>Poettering</surname>
-                                <email>lennart@poettering.net</email>
-                        </author>
-                </authorgroup>
-        </refentryinfo>
-
-        <refmeta>
-                <refentrytitle>systemd-loginctl</refentrytitle>
-                <manvolnum>1</manvolnum>
-        </refmeta>
-
-        <refnamediv>
-                <refname>systemd-loginctl</refname>
-                <refpurpose>Control the systemd login manager</refpurpose>
-        </refnamediv>
-
-        <refsynopsisdiv>
-                <cmdsynopsis>
-                        <command>systemd-loginctl <arg choice="opt" rep="repeat">OPTIONS</arg> <arg choice="req">COMMAND</arg> <arg choice="opt" rep="repeat">NAME</arg></command>
-                </cmdsynopsis>
-        </refsynopsisdiv>
-
-        <refsect1>
-                <title>Description</title>
-
-                <para><command>systemd-loginctl</command> may be used to
-                introspect and control the state of the
-                <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-                login manager.</para>
-        </refsect1>
-
-        <refsect1>
-                <title>Options</title>
-
-                <para>The following options are understood:</para>
-
-                <variablelist>
-                        <varlistentry>
-                                <term><option>--help</option></term>
-                                <term><option>-h</option></term>
-
-                                <listitem><para>Prints a short help
-                                text and exits.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--version</option></term>
-
-                                <listitem><para>Prints a short version
-                                string and exits.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--property=</option></term>
-                                <term><option>-p</option></term>
-
-                                <listitem><para>When showing
-                                session/user/ properties, limit
-                                display to certain properties as
-                                specified as argument. If not
-                                specified all set properties are
-                                shown. The argument should be a
-                                property name, such as
-                                <literal>Sessions</literal>. If
-                                specified more than once all
-                                properties with the specified names
-                                are shown.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--all</option></term>
-                                <term><option>-a</option></term>
-
-                                <listitem><para>When showing
-                                unit/job/manager properties, show all
-                                properties regardless whether they are
-                                set or not.</para></listitem>
-                        </varlistentry>
-
-
-                        <varlistentry>
-                                <term><option>--no-pager</option></term>
-
-				<listitem><para>Do not pipe output into a
-				pager.</para></listitem>
-			</varlistentry>
-
-                        <varlistentry>
-                                <term><option>--kill-who=</option></term>
-
-                                <listitem><para>When used with
-                                <command>kill-session</command>,
-                                choose which processes to kill. Must
-                                be one of <option>leader</option>, or
-                                <option>all</option> to select whether
-                                to kill only the leader process of the
-                                session or all processes of the
-                                session. If omitted defaults to
-                                <option>all</option>.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>--signal=</option></term>
-                                <term><option>-s</option></term>
-
-                                <listitem><para>When used with
-                                <command>kill-session</command> or
-                                <command>kill-user</command>, choose
-                                which signal to send to selected
-                                processes. Must be one of the well
-                                known signal specifiers such as
-                                SIGTERM, SIGINT or SIGSTOP. If omitted
-                                defaults to
-                                <option>SIGTERM</option>.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>-H</option></term>
-                                <term><option>--host</option></term>
-
-                                <listitem><para>Execute operation
-                                remotely. Specify a hostname, or
-                                username and hostname separated by @,
-                                to connect to. This will use SSH to
-                                talk to the remote login manager
-                                instance.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><option>-P</option></term>
-                                <term><option>--privileged</option></term>
-
-                                <listitem><para>Acquire privileges via
-                                PolicyKit before executing the
-                                operation.</para></listitem>
-                        </varlistentry>
-                </variablelist>
-
-                <para>The following commands are understood:</para>
-
-                <variablelist>
-                        <varlistentry>
-                                <term><command>list-sessions</command></term>
-
-                                <listitem><para>List current sessions.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>session-status [ID...]</command></term>
-
-                                <listitem><para>Show terse runtime
-                                status information about one or more
-                                sessions. This function is intended to
-                                generate human-readable output. If you
-                                are looking for computer-parsable
-                                output, use
-                                <command>show-session</command>
-                                instead.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>show-session [ID...]</command></term>
-
-                                <listitem><para>Show properties of one
-                                or more sessions or the manager
-                                itself. If no argument is specified
-                                properties of the manager will be
-                                shown. If a session ID is specified
-                                properties of the session is shown. By
-                                default, empty properties are
-                                suppressed. Use <option>--all</option>
-                                to show those too. To select specific
-                                properties to show use
-                                <option>--property=</option>. This
-                                command is intended to be used
-                                whenever computer-parsable output is
-                                required. Use
-                                <command>session-status</command> if
-                                you are looking for formatted
-                                human-readable
-                                output.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>activate [ID...]</command></term>
-
-                                <listitem><para>Activate one or more
-                                sessions. This brings one or more
-                                sessions into the foreground, if
-                                another session is currently in the
-                                foreground on the respective
-                                seat.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>lock-session [ID...]</command></term>
-                                <term><command>unlock-session [ID...]</command></term>
-
-                                <listitem><para>Activates/deactivates
-                                the screen lock on one or more
-                                sessions, if the session supports it.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>terminate-session [ID...]</command></term>
-
-                                <listitem><para>Terminates a
-                                session. This kills all processes of
-                                the session and deallocates all
-                                resources attached to the
-                                session.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>kill-session [ID...]</command></term>
-
-                                <listitem><para>Send a signal to one
-                                or more processes of the session. Use
-                                <option>--kill-who=</option> to select
-                                which process to kill. Use
-                                <option>--signal=</option> to select
-                                the signal to send.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>list-users</command></term>
-
-                                <listitem><para>List currently logged
-                                in users.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>user-status [USER...]</command></term>
-
-                                <listitem><para>Show terse runtime
-                                status information about one or more
-                                logged in users. This function is
-                                intended to generate human-readable
-                                output. If you are looking for
-                                computer-parsable output, use
-                                <command>show-user</command>
-                                instead. Users may be specified by
-                                their usernames or numeric user
-                                IDs.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>show-user [USER...]</command></term>
-
-                                <listitem><para>Show properties of one
-                                or more users or the manager
-                                itself. If no argument is specified
-                                properties of the manager will be
-                                shown. If a user is specified
-                                properties of the user is shown. By
-                                default, empty properties are
-                                suppressed. Use <option>--all</option>
-                                to show those too. To select specific
-                                properties to show use
-                                <option>--property=</option>. This
-                                command is intended to be used
-                                whenever computer-parsable output is
-                                required. Use
-                                <command>user-status</command> if
-                                you are looking for formatted
-                                human-readable
-                                output.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>enable-linger [USER...]</command></term>
-                                <term><command>disable-linger [USER...]</command></term>
-
-                                <listitem><para>Enable/disable user
-                                lingering for one or more users. If
-                                enabled for a specific user a user
-                                manager is spawned for him/her at
-                                boot, and kept around after
-                                logouts. This allows users who aren't
-                                logged in to run long-running
-                                services.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>terminate-user [USER...]</command></term>
-
-                                <listitem><para>Terminates all
-                                sessions of a user. This kills all
-                                processes of all sessions of the user
-                                and deallocates all runtime resources
-                                attached to the
-                                user.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>kill-user [USER...]</command></term>
-
-                                <listitem><para>Send a signal to all
-                                processes of a user. Use
-                                <option>--signal=</option> to select
-                                the signal to send.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>list-seats</command></term>
-
-                                <listitem><para>List currently
-                                available seats on the local
-                                system.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>seat-status [NAME...]</command></term>
-
-                                <listitem><para>Show terse runtime
-                                status information about one or more
-                                seats. This function is
-                                intended to generate human-readable
-                                output. If you are looking for
-                                computer-parsable output, use
-                                <command>show-seat</command>
-                                instead.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>show-seat [NAME...]</command></term>
-
-                                <listitem><para>Show properties of one
-                                or more seats or the manager
-                                itself. If no argument is specified
-                                properties of the manager will be
-                                shown. If a seat is specified
-                                properties of the seat are shown. By
-                                default, empty properties are
-                                suppressed. Use <option>--all</option>
-                                to show those too. To select specific
-                                properties to show use
-                                <option>--property=</option>. This
-                                command is intended to be used
-                                whenever computer-parsable output is
-                                required. Use
-                                <command>seat-status</command> if you
-                                are looking for formatted
-                                human-readable
-                                output.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>attach [NAME] [DEVICE...]</command></term>
-
-                                <listitem><para>Attach one or more
-                                devices to a seat. The devices should
-                                be specified via device paths in the
-                                <filename>/sys</filename> file
-                                system. To create a new seat attach at
-                                least one graphics card to a
-                                previously unused seat names. seat
-                                names may consist only of a-z, A-Z,
-                                0-9, "-" and "_" and must be prefixed
-                                with "seat". To drop assignment of a
-                                device to a specific seat just
-                                reassign it to a different seat, or
-                                use
-                                <command>flush-devices</command>.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>flush-devices</command></term>
-
-                                <listitem><para>Removes all device
-                                assignments previously created with
-                                <command>attach</command>. After this
-                                call only automatically generated
-                                seats will remain and all seat
-                                hardware is assigned to
-                                them.</para></listitem>
-                        </varlistentry>
-
-                        <varlistentry>
-                                <term><command>terminate-seat [NAME...]</command></term>
-
-                                <listitem><para>Terminates all
-                                sessions on a seat. This kills all
-                                processes of all sessions on a seat and
-                                deallocates all runtime resources
-                                attached to them.</para></listitem>
-                        </varlistentry>
-                </variablelist>
-
-        </refsect1>
-
-        <refsect1>
-                <title>Exit status</title>
-
-                <para>On success 0 is returned, a non-zero failure
-                code otherwise.</para>
-        </refsect1>
-
-        <refsect1>
-                <title>Environment</title>
-
-                <variablelist>
-                        <varlistentry>
-                                <term><varname>$SYSTEMD_PAGER</varname></term>
-                                <listitem><para>Pager to use when
-                                <option>--no-pager</option> is not given;
-                                overrides <varname>$PAGER</varname>.  Setting
-                                this to an empty string or the value
-                                <literal>cat</literal> is equivalent to passing
-                                <option>--no-pager</option>.</para></listitem>
-                        </varlistentry>
-                </variablelist>
-        </refsect1>
-
-        <refsect1>
-                <title>See Also</title>
-                <para>
-                        <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemd-logind.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-                </para>
-        </refsect1>
-
-</refentry>
diff --git a/man/systemd-logind.conf.xml b/man/systemd-logind.conf.xml
index 4b02815..27c11c2 100644
--- a/man/systemd-logind.conf.xml
+++ b/man/systemd-logind.conf.xml
@@ -167,7 +167,7 @@
                   <title>See Also</title>
                   <para>
                           <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd-loginctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>loginctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
                   </para>
         </refsect1>
diff --git a/man/systemd.exec.xml b/man/systemd.exec.xml
index ac0f89f..e6f49c9 100644
--- a/man/systemd.exec.xml
+++ b/man/systemd.exec.xml
@@ -397,7 +397,7 @@
                                 <citerefentry><refentrytitle>dmesg</refentrytitle><manvolnum>1</manvolnum></citerefentry>. <option>journal</option>
                                 connects it with the journal which is
                                 accessible via
-                                <citerefentry><refentrytitle>systemd-journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+                                <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
                                 (Note that everything that is written
                                 to syslog or kmsg is implicitly stored
                                 in the journal as well, those options
@@ -1094,6 +1094,7 @@
                   <para>
                           <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.socket</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-- 
1.7.9.3

